### PR TITLE
Automate qualification from master to stable.

### PR DESCRIPTION
**Pull request by sebastienvas**
Added scripts and libaries to help qualification from master branch (head) to stable (base). The go program will get information from github API to make sure checks have passed and will move the stable to the master commit in that PR.

### CODE REVIEW

**Comment by sebastienvas** [comment]
@kyessenov Could you please review the go file and provide feedback.

**Comment by sebastienvas** [comment]
#13

File: resources/github_pr.go [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+var (
+	tokenFile   = flag.String("token_file", "", "File containing Auth Token.")
+	owner       = flag.String("owner", "istio", "Github Owner or org.")
+	repo        = flag.String("repo", "", "Github repo within the org.")
+	base        = flag.String("base", "stable", "The base branch used for PR.")
+	head        = flag.String("head", "master", "The head branch used for PR.")
+	fastForward = flag.Bool("fast_forward", false, "Creates a PR updating Base to Head.")
+	verify      = flag.Bool("verify", false, "Verifies PR on Base and push them if success.")
+	GH          = newGhConst()
```
Comment (kyessenov) [inline resources/github_pr.go:24]: Why not define a bunch of const strings?

File: resources/github_pr.go [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+	gh.all = "all"
+	gh.commit = "commit"
+	return gh
+}
+
+// Creates a new Github Helper from provided
+func newHelper() (*helper, error) {
+	if tc, err := getToken(); err == nil {
+		client := github.NewClient(tc)
+		h := new(helper)
```
Comment (kyessenov) [inline resources/github_pr.go]: &helper{ Owner: .... } looks better

File: resources/github_pr.go (hunk 1 above)
Comment (sebastienvas) [reply resources/github_pr.go:24]: That s what I did at the beginning and for some of the struct require string pointers which made the rest of the code ugly.

File: resources/github_pr.go (hunk 2 above)
Comment (sebastienvas) [reply resources/github_pr.go]: Done

**Review by kyessenov — APPROVED** [review]

**Pull request merged**